src/foam/u2/crunch/CapabilityInterceptView.js

```diff
@@ -65,31 +103,40 @@ foam.CLASS({
       var self = this;
       this
         .addClass(this.myClass())
-        .start(this.Rows)
-          .addClass(this.myClass('detail-container'))
-          .add(this.slot(function (data$capabilityOptions) {
-            return this.E().select(this.capabilityDAO.where(
-              self.IN(self.Capability.ID, data$capabilityOptions)
-            ), (cap) => {
-              return this.E().tag(self.CapabilityCardView, {
-                data: cap
-              })
-                .on('click', () => {
-                  var p = self.crunchController.launchWizard(cap);
-                  p.then(() => {
-                    this.checkStatus(cap);
-                  })
+        .start('h1').add(this.TITLE).end()
+        .start('h4').add(this.SUBTITLE_1).end()
+        .start('h4').add(this.SUBTITLE_2).end()
+        .start().addClass(this.myClass('mainSection'))
+          .start(this.Rows)
+            .addClass(this.myClass('detail-container'))
+            .start('h3').add(this.CAP_TITLE).end()
+            .add(this.slot(function(data$capabilityOptions) {
+              return this.E().select(this.capabilityDAO.where(
+                self.IN(self.Capability.ID, data$capabilityOptions)
+              ), cap => {
+                return this.E().tag(self.CapabilityCardView, {
+                  data: cap
                 })
-            })
-          }))
+                  .on('click', () => {
+                    var p = self.crunchController.launchWizard(cap.id);
+                    p.then(() => {
+                      this.checkStatus(cap);
+                    });
+                  });
+              }).addClass(this.myClass('capList-css'));
+            }))
+          .end()
+          .start().addClass(this.myClass('legendSize')).tag(foam.u2.view.EnumLegendView, { of: foam.nanos.crunch.CapabilityJunctionStatus, enumValueToHide: ['APPROVED'] }).end()
         .end()
         .startContext({ data: this })
           .tag(this.CANCEL, { buttonStyle: 'SECONDARY' })
         .endContext();
     },
+
     function checkStatus(cap) {
```

Comment:
Now that CrunchService exists a call to `this.crunchService.getJunction(null, cap.id)` is better. We currently have this logic duplicated in a lot of places so IMO we should work on getting those to use the CrunchService method also.

Reply:
adjustments coming